card/lib/card/name.rb

```diff
@@ -37,5 +37,9 @@ def trait tag_code
       name = trait_name( tag_code )
       name ? name.s : ( raise Card::NotFound, "unknown codename: #{tag_code}" )
     end
+    
+    def code
+      Codename[ Card.fetch_id self ]
+    end
```

Comment:
yeah, this is pretty much exactly how I was thinking of it.  Probably a good idea, not a huge deal.